AmahiAnywhere/AmahiAnywhere/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>735481446010-va9kissnqfi5d227fcdhh9epdu827hhd.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.735481446010-va9kissnqfi5d227fcdhh9epdu827hhd</string>
+	<key>API_KEY</key>
+	<string>AIzaSyCtPHp_pu1rr3xx1dUHq9wazMny4gv-QDo</string>
+	<key>GCM_SENDER_ID</key>
+	<string>735481446010</string>
+	<key>PLIST_VERSION</key>
+	<string>1</string>
+	<key>BUNDLE_ID</key>
+	<string>com.amahi.AmahiAnywhere</string>
+	<key>PROJECT_ID</key>
+	<string>amahi-android</string>
+	<key>STORAGE_BUCKET</key>
+	<string>amahi-android.appspot.com</string>
+	<key>IS_ADS_ENABLED</key>
+	<false></false>
+	<key>IS_ANALYTICS_ENABLED</key>
+	<false></false>
+	<key>IS_APPINVITE_ENABLED</key>
+	<true></true>
+	<key>IS_GCM_ENABLED</key>
+	<true></true>
+	<key>IS_SIGNIN_ENABLED</key>
+	<true></true>
```

Comment:
what does this imply??

Reply:
@Cpg This GoogleService-Info.plist file contains configuration details, such as keys and identifiers, for the services that you have just enabled so we can't add or subtract anything from this file.